ngraph/test/serialize.cpp

```diff
@@ -144,21 +144,6 @@ TEST(serialize, friendly_name)
 }
 #endif
 
-TEST(serialize, existing_models)
```

Comment:
This test has nothing to do with the Add operation. Why is it removed?

Reply:
Models from this test cannot be serialised, because model contains v0::Add op.